modules/widgets/top-posts.php

```diff
@@ -445,6 +445,9 @@ function get_by_views( $count, $args ) {
 
 		/**
 		 * Filter the number of days used to calculate Top Posts for the Top Posts widget.
+		 * We do not recommend accessing more than 10 days of results at one.
```

Comment:
Could you add an empty line between the filter title and the description? Our parser likes that better, when outputting documentation.